src/queries/rotate-domainkeys.sql

```diff
@@ -23,14 +23,15 @@ IF EXISTS (
     AND readonly = FALSE
 ) THEN
   SET newkey = IF(@newkey = "-", GENERATE_UUID(), @newkey);
-  CALL helix_rum . ROTATE_DOMAIN_KEYS( -- noqa: PRS
+  CALL helix_reporting . ROTATE_DOMAIN_KEYS( -- noqa: PRS
```

Comment:
```suggestion
  CALL helix_reporting.ROTATE_DOMAIN_KEYS( -- noqa: PRS
```

I know this is SQLFluffs doing, but sometimes you can coax it back to reasonable formatting.

Reply:
Change made, needed an additional noqa rule.